stable_diffusion/stable_diffusion/model_io.py

```diff
@@ -140,6 +140,17 @@ def map_vae_weights(key, value):
     if "to_v" in key:
         key = key.replace("to_v", "value_proj")
 
+
+    # Map attention layers in SD-2-1-base:VAE
+    if "key" in key:
+        key = key.replace("key", "to_k")
+    if "proj_attn" in key:
+        key = key.replace("proj_attn", "out_proj")
+    if "query" in key:
+        key = key.replace("query", "query_proj")
+    if "value" in key:
+        key = key.replace("value", "value_proj")
```

Comment:
These don't look correct given that they will result in `value_proj_proj` and `query_proj_proj` and `to_k`. Am I missing sth?